docs/try/tutorials/geomodel/geomodel-basics.md

```diff
@@ -1,22 +1,76 @@
 
-## How to build and use GeoModel
-### GeoModelBuilder
+## Basic functionalities on GeoModel
+The most efficient way to build a GeoModel is to import its geometry and topology from a
+geomodeler. This tutorial present a simple code to load manipulate and export `GeoModel`.
+
+### GeoModel I/O
+
+GeoModel stores the geometry and the topology of a geological model. It can be imported from boundary 
+representation or volumetric representation file format.
+RINGMesh uses an extensible design to handle various file formats.
+The file extention is the key for selecting the suitable loader/saver. 
+The Input/Output file formats currently supported by RINGMesh are listed [here](/features/file_formats).
+
+A GeoModel can be loaded and saved as follow:
+
+	 // GeoModel instantiation in 3D
```

Comment:
I think you can use a markdown syntax to improve code readibility using ```cpp at he begining of the code block and ``` at the end.

Reply:
maybe you can remove the cpp flag, I do not know what mkdocs will do.